l1-contracts/test/foundry/unit/concrete/Bridgehub/experimental_bridge.t.sol

```diff
@@ -641,8 +879,15 @@ contract ExperimentalBridgeTest is Test {
         mockChainContract.setBridgeHubAddress(address(bridgeHub));
         assertTrue(mockChainContract.getBridgeHubAddress() == address(bridgeHub));
 
-        vm.txGasPrice(0.05 ether);
+        if (msgValue != mockMintValue) {
```

Comment:
can it be `if(msgValue < mockMintValue)` instead of the strict equality used here?

Reply:
Require statements has strict equality, so I guess strict inequality fits here. I've separated success/reverts, moved revert into separate function and used vm.assume to discard equal values.